setup.py

```diff
@@ -21,8 +11,25 @@
     license='GPLv3+',
     packages=find_packages(exclude=['tests']),
     include_package_data=True,
-    install_requires=requirements,
-    dependency_links=dependency_links,
+    install_requires=[
+        'flask',
+        'future',
+        'wheel',
```

Comment:
What is this for?

Reply:
When I was building image locally, it wasn't building without those deps.

Reply:
And btw, `flask` and `future` required by estuary, so they will be in dependencies enyway. I can delete them, but I don't remember why I added `wheel`, I will delete it too then.